server.js

```diff
@@ -19,13 +20,18 @@ const middleware = [
   require('./lib/middleware/authentication/authentication.js')(),
   require('./lib/middleware/extensions/extensions.js')
 ]
-const config = require(`${process.cwd()}/app/config.js`)
+const config = require('./lib/config.js')
 const prototypeAdminRoutes = require('./lib/prototype-admin-routes.js')
 const packageJson = require('./package.json')
-const routes = require(`${process.cwd()}/app/routes.js`)
+const routesPath = `${process.cwd()}/app/routes.js`
```

Comment:
Can we use `projectDir` here?

```suggestion
const routesPath = `${projectDir}/app/routes.js`
```